packages/carousel/src/ArrowButton/ArrowButton.js

```diff
@@ -27,6 +27,8 @@ ArrowButton.defaultProps = {
   buttonColor: 'primary.base',
   buttonHoverBackground: 'background.base',
   buttonHoverColor: 'primary.dark',
+  buttonHeight: '60px',
```

Comment:
I removed `buttonHeight` and `buttonWidth` and replaced it with `buttonSize` but kept the type as string and `sideButtonMargin` flexible because in the end making the change would mean i would have to pass buttonSize to the Carousel component and the passed in ArrowButton component `<Carousel buttonSize={48} sidePositionArrowButton={<ArrowButton buttonSize={48} />}` and wanted to avoid the duplication.

Reply:
Sounds good